File: Day-2/Ques8.py
num = int(input("Enter a number:"))
original = num
rev = 0

while num > 0:
    digit = num % 10
    rev = rev * 10 + digit
    num //= 10

if original == rev:
    print ("palindrome number")
else:
    print("Not a palindrome number")

File: Day-2/Ques8.c
#include <stdio.h>
 int main() {
    int num,n1,digit,rev=0;
    printf("Enter a number:");
    scanf("%d", &num);

    n1 = num;

    while (num >0){
        digit = num % 10;
        rev = rev * 10 + digit;
        num /= 10;
    }
    if (n1==rev){
        printf("The number is palindrome");
    }
    else{
        printf("The number is not a palindrome");
    }
    return 0;
}